Automattic/jetpack
# Blocks: Convert SVG attributes names to camelCase to make React happy

#### Changes proposed in this Pull Request:

* Convert several SVG attributes names to camelCase.

React expects props (and HTML attributes) names to be in camelCase, but SVG files are normally created as XML files, which wants attributes names in kebab-case.
React then throws console errors about them. E.g.
```
Warning: Invalid DOM property `stroke-width`. Did you mean `strokeWidth`?
```
These errors should be suppressed in the production build, and don't compromise the functionality of the app (kebab-case property should be used "as-is", so in this case they work fine), but create a lot of unnecessary console noise when working on the block editor.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* If you're an Automattician, include a shortlink to the p2 discussion with Jetpack Product here.

#### Testing instructions:






* Enter the block editor and open the browser console.
* Open the block inserter (e.g. by typing <kbd>/</kbd>).
* Make sure there are no `Warning: Invalid DOM property` errors in the console.

#### Proposed changelog entry for your changes:

* Blocks: Fix SVG attributes naming.
